erna/scripts/process_fact_mc.py

```diff
@@ -15,9 +15,25 @@
 
 logger = logging.getLogger(__name__)
 
-
+import re
+
+# given a string create a filename
+def createFilenameFromFormat(format, basename, num):
```

Comment:
do not use lower camel case for function names in python, use `snake_case`

Reply:
Do not use `format`, its a python buildin.

Documentation what this function does!